src/utils.ts

```diff
@@ -27,14 +25,15 @@ const getBrowser = (userAgent) => {
         browser = 'safari';
     }
 
-    return browser;
+    return browser as BrowserSpecification['name'];
 };
 
-const getBrowsersWithVersions = (browserList) => {
-    const browsersWithVersions = {};
+const getBrowsersWithVersions = (browserList: string[]): Record<BrowserSpecification['name'], number[]> => {
+    const browsersWithVersions: Record<string, number[]> = {};
 
     for (const browserDefinition of browserList) {
-        const [browser, versionString] = browserDefinition.split(' ');
+        const [browserSplit, versionString] = browserDefinition.split(' ');
+        const browser = browserSplit as BrowserSpecification['name'];
```

Comment:
no changes needed, just note that this might work too
```suggestion
        const [browser, versionString] = browserDefinition.split(' ') as [BrowserSpecification['name'], string];
```

Reply:
Nice! :)